dask/dataframe/core.py

```diff
@@ -7330,7 +7330,10 @@ def _cov_corr_chunk(df, corr=False):
         mask = df.iloc[:, idx].notnull()
         sums[idx] = df[mask].sum().values
         counts[idx] = df[mask].count().values
-    cov = df.cov().values
+    # pandas will produce warnings on a single-row dataframe, but we don't want to fail there
+    with warnings.catch_warnings():
+        warnings.filterwarnings("ignore", category=RuntimeWarning)
+        cov = df.cov().values
```

Comment:
You want to suppress all warnings here?

Reply:
I'm not sure what's the best to do here. If you look at my example in the description, it produces 3 different warning messages, and I don't know if it may produce some other ones, depending on data. Filtering 3 different warnings seems ugly. What do you think?

Reply:
That's ok with me, just wanted to make sure that I understood you correctly. We could also pass the warnings to the user, but not sure if this makes sense here?

Reply:
Yeah this is not a useful warning. User can't do anything about it.